javascript/javascript1/week4/h4.js

```diff
@@ -0,0 +1,97 @@
+let userName = "";
+const toDoList = [];
+
+function getReply(command) {
+    if (typeof command !=='string' || command === "") {
+        return `Error! Not a valide command`;
+    } if (command === "Hello my name is Violeta" && userName === "") {
+        userName = "Violeta";
+        return `Nice to meet you ${userName}`;
+    } else if (command === "Hello my name is Violeta" && userName === ""){
+        userName="";
+        return `Your name is already added`;
```

Comment:
You should check what is written after `Hello my name is`, and compare it with `userName`. Only if equal, you should `return "Your name is already added";` and there is no need to change `userName`

Reply:
Good reading when to use backticks https://blog.bitsrc.io/double-quotes-vs-single-quotes-vs-backticks-in-javascript-3cab5aaea55